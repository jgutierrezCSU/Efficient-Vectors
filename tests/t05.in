areIdentical
10 10
2	3	11	11	19	32	34	37	38	47
2	3	11	11	19	32	34	37	38	47
areIdentical
10 10
2	3	11	11	19	32	34	37	38	47
2	3	11	11	19	25	34	37	38	47
areIdentical
10 8
2	3	11	11	19	32	34	37	38	47
2	3	11	11	19	32	34	37
areIdentical
50 50
1	2	2	3	3	6	6	7	8	10	13	14	15	15	15	17	17	17	18	18	19	19	19	20	20	20	21	23	25	28	29	30	33	33	34	35	38	39	40	41	42	43	43	44	44	45	45	48	50	50
1	2	2	3	3	6	6	7	8	10	13	14	15	15	15	17	17	17	18	18	19	19	19	20	20	20	21	22	25	28	29	30	33	33	34	35	38	39	40	41	42	43	43	44	44	45	45	48	50	50
areIdentical
100 100
342164	172299	484418	330015	58098	189363	263275	440741	274061	132135	178514	128389	278230	83379	347755	285418	23373	7335	242068	427520	361621	432081	12135	483165	382929	225119	110094	454619	427016	234796	313454	60229	34446	399207	456138	466470	393549	65865	253043	484734	390158	257783	193750	201274	463756	122693	367101	211321	353843	492985	420572	458707	212014	346397	239125	422119	44594	208124	325947	223884	18630	146296	314765	193024	296800	257565	448840	496154	107830	334488	366257	393439	293502	121095	9379	1922	174907	333347	334253	149160	55946	35997	117978	83872	406691	42630	150963	336371	88960	248319	483478	303975	225875	244731	491486	398857	210138	326564	452994	124789
342164	172299	484418	330015	58098	189363	263275	440741	274061	132135	178514	128389	278230	83379	347755	285418	23373	7335	242068	427520	361621	432081	12135	483165	382929	225119	110094	454619	427016	234796	313454	60229	34446	399207	456138	466470	393549	65865	253043	484734	390158	257783	193750	201274	463756	122693	367101	211321	353843	492985	420572	458707	212014	346397	239125	422119	44594	208124	325947	223884	18630	146296	314765	193024	296800	257565	448840	496154	107830	334488	366257	393439	293502	121095	9379	1922	174907	333347	334253	149160	55946	35997	117978	83872	406691	42630	150963	336371	88960	248319	483478	303975	225875	244731	491486	398857	210138	326564	452994	124789

