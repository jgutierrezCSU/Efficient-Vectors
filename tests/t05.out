Size of A is 10, size of B is 10, testing whether A and B are identical:
A and B are identical.
Size of A is 10, size of B is 10, testing whether A and B are identical:
A and B are NOT identical.
Size of A is 10, size of B is 8, testing whether A and B are identical:
A and B are NOT identical.
Size of A is 50, size of B is 50, testing whether A and B are identical:
A and B are NOT identical.
Size of A is 100, size of B is 100, testing whether A and B are identical:
A and B are identical.
